test/GithubApi.Users.test.js

```diff
@@ -0,0 +1,43 @@
+const axios = require('axios');
+const chai = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const { expect } = chai;
+chai.use(require('chai-subset'));
+
+const urlBase = 'https://api.github.com/users';
+const keyPath = axios.create({
+  path: urlBase,
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+
+let params = {
+  per_page: 10
+};
+
+describe('Stage 12 - Query parameters: Users', () => {
+  let users;
+  before(async () => {
+    users = await keyPath.get(`${urlBase}`, { params });
+  });
+
```

Comment:
You are missing one test, that is the one without params to verify that it returns 30 records by default